### PR TITLE
feat: add textarea

### PR DESCRIPTION
**Pull request by b-avb**

### CODE REVIEW

File: src/components/atoms/textarea.rs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            if let Some(value) = cx.props.label {
+                rsx!(
+                    label {
+                        class: "input__label",
+                        "{value}"
+                    }
+                )
+            }
+            div {
+                id: "textarea-wrapper",
```
Comment (olanod) [inline src/components/atoms/textarea.rs:38]: As an atom that can have multiple instances it shouldn't have an `id` right?

File: src/components/atoms/textarea.rs [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                class: "input-wrapper",
+                textarea {
+                    id: "textarea",
```
Comment (olanod) [inline src/components/atoms/textarea.rs:41]: ☝🏽

**Review by olanod — APPROVED** [review]
Minor comment, otherwise nice :) Looks good to me 👍🏽

**Comment by b-avb** [comment]
Now we do not use the ids. With web_sys and wasm_bindgen we achieve that when mounting the element we can reference and mutate it.

**Review by olanod — APPROVED** [review]
_Looks Good To Me_! 👌🏽, for future size related optimizations we can try to take out as much of the inline styles as possible(into separate stylesheets) and deal mainly with toggling of css classes, all the string literals in the code base end up taking quite some space in the final blob.

**Pull request merged**